src/components/custom-media-library/ContentEmpty.tsx

```diff
@@ -0,0 +1,27 @@
+import { CSSProperties } from "react";
+import { FormattedMessage } from "react-intl";
+
+const containerStyle: CSSProperties = {
+  alignItems: "center",
+  display: "flex",
+  flexDirection: "column",
+  justifyContent: "center",
+  textAlign: "center",
+  padding: "60px",
+};
+
+const textStyle: CSSProperties = { color: "#666" };
```

Comment:
These styles are temporal, we should work with @GusBaamonde to move them to the Style Guide.